tests/lib/rules/no-console.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * @author ItMaga <https://github.com/ItMaga>
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const RuleTester = require('eslint').RuleTester
+const rule = require('../../../lib/rules/no-console')
+
+const tester = new RuleTester({
+  parser: require.resolve('vue-eslint-parser'),
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module'
+  }
+})
+
+tester.run('no-console', rule, {
+  valid: [
+    {
+      filename: 'test.vue',
+      code: `
+      <template>
+        <button @click="Console.log">button</button>
+      </template>
+      `,
+      options: [{ allow: ['log'] }]
```

Comment:
I don't think this option is needed if the intent was to test when the identifiers don't match.

```suggestion
```

Reply:
Done